django/db/models/query.py

```diff
@@ -541,7 +541,7 @@ def bulk_update(self, objs, fields, batch_size=None):
         if any(f.primary_key for f in fields):
             raise ValueError('bulk_update() cannot be used with primary key fields.')
         if not objs:
```

Comment:
In comparison to initializing a variable here for just returning 0 when it would be essentially used in the latter part of the code v/s initializing it where it is used, I think the latter seems more readable to me. What do you say?

Reply:
@abhiabhi94 Agreed, let's leave `return 0` as a micro-optimization.

Reply:
thanks for taking my side 😆